test/puppetlabs/puppetdb/cli/import_export_roundtrip_test.clj

```diff
@@ -14,23 +18,25 @@
             [clj-time.coerce :refer [to-string]]
             [puppetlabs.puppetdb.utils :as utils]
             [puppetlabs.puppetdb.client :as pdb-client]
+            [puppetlabs.puppetdb.helpers :as helpers]
             [slingshot.slingshot :refer [throw+ try+]]
             [slingshot.test]
-            [puppetlabs.puppetdb.testutils.services :as svc-utils :refer [*base-url*]]))
+            [puppetlabs.puppetdb.testutils.services :as svc-utils]))
 
 (use-fixtures :each fixt/with-test-logging-silenced)
 
 (defn munge-report
   [report]
-  (map (comp tur/munge-report-for-comparison tur/munge-example-report-for-storage)
+  (map (comp tur/munge-report-for-comparison
+             tur/munge-example-report-for-storage)
        (-> report
            utils/vector-maybe)))
 
 (defn munge-catalog
   [catalog]
-  (map (partial tuc/munge-catalog-for-comparison :v6)
+  (map (comp (partial tuc/munge-catalog-for-comparison :v6)
+             #(dissoc % :hash))
```

Comment:
should it be renamed to call that out? also it looks like there's no longer a benefit to the -> below